agent/bench-scripts/pbench-fio.md

```diff
@@ -200,21 +200,21 @@ Then you construct a fio job file for your initial sequential tests, and this wi
 
     [global]
     # size of each FIO file (in MB)
-    size=$@
+    size=1GiB
     # do not use gettimeofday, much too expensive for KVM guests
     clocksource=clock_gettime
     # give fio workers some time to get launched on remote hosts
     startdelay=5
     # files accessed by fio are in this directory
-    directory=$@
+    directory=/mnt/fio/files
```

Comment:
Not sure about this change either, see our basic job templates:

 * https://github.com/distributed-system-analysis/pbench/blob/master/agent/bench-scripts/templates/fio.job
 * https://github.com/distributed-system-analysis/pbench/blob/master/agent/bench-scripts/templates/fio-shared-fs.job

The `fio-shared-fs.job` template uses `directory` with the `$target` environment variable reference that allows us to pass that from the command line.

Reply:
With the command line I've used, this parameter is not getting filled. Also the pdsh commands above assumes the directory path is fixed.